packages/sanity/src/core/components/inputs/DateInputs/calendar/Calendar.tsx

```diff
@@ -106,9 +88,9 @@ export const Calendar = forwardRef(function Calendar(
     ...restProps
   } = props
 
-  const focusedDate = _focusedDate ?? savedSelectedDate
-
-  const {timeZone, zoneDateToUtc} = useTimeZone(timeZoneScope)
+  const {timeZone, zoneDateToUtc, utcToCurrentZoneDate} = useTimeZone(timeZoneScope)
+  const currentTzDate = useMemo(() => utcToCurrentZoneDate(value), [utcToCurrentZoneDate, value])
+  const [focusedDate, setFocusedDate] = useState<Date>(value)
```

Comment:
interest, I see that we are moving from seting the focusedDate as the _focusedDate / savedSelctedDate and instead focus on value. 
In general we are removing the set focused date? I'm just wondering what ramifications this might have